### PR TITLE
Run brodocs container as the current user.

### LINKED ISSUE #202: "kubebuilder docs" creates a "build" directory whose owner is root
The owner should be the current user, not root. Currently, after running `kubebuilder docs`, I have to do a `chown` for that directory in order to be able to continue the workflow.

### PR DESCRIPTION
**Pull request by pwittrock**
Fixes #202

### CODE REVIEW

File: cmd/kubebuilder/docs/docs.go [hunk 1]
```diff
@@ -157,13 +158,21 @@ func RunDocs(cmd *cobra.Command, args []string) {
...
 		c = exec.Command("docker", "run",
+			"-u", fmt.Sprintf("%s:%s", user.Uid, user.Gid),
+			"-v", "/etc/group:/etc/group:ro",
+			"-v", "/etc/passwd:/etc/passwd:ro",
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir, "includes"), "/source"),
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir, "build"), "/build"),
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir, "build"), "/build"),
 			"-v", fmt.Sprintf("%s:%s", filepath.Join(wd, outputDir), "/manifest"),
-			"gcr.io/kubebuilder/brodocs",
+			"gcr.io/kubebuilder/brodocs:v0.1.11",
```
Comment (kragniz) [inline cmd/kubebuilder/docs/docs.go]: should this version tag be hard coded?

Comment (pwittrock) [reply cmd/kubebuilder/docs/docs.go]: Yes I think so.  It makes it so we don't need to test each brodocs version with each kubebuilder release.  If we just use latest, we would have to test each brodocs image with each kubebuilder release, or risk breaking old kubebuilder versions.

Comment (kragniz) [reply cmd/kubebuilder/docs/docs.go]: I was thinking more along the lines of using something like kubeBuilderVersion from cmd/kubebuilder/version, falling back to latest if that's not defined

Comment (kragniz) [reply cmd/kubebuilder/docs/docs.go]: I guess something that could be done after this patch

**Pull request merged**